locuszoom.app.js

```diff
@@ -267,6 +267,11 @@ LocusZoom.createCORSPromise = function (method, url, body, timeout) {
         };
         timeout && setTimeout(response.reject, timeout);
         body = typeof body !== "undefined" ? body : "";
+        // If posting an object set the proper content-type header
+        if (method == "POST"){
+            xhr.setRequestHeader("Content-Type","application/x-www-form-urlencoded");
```

Comment:
This wouldn't be the right content type for all POSTs. Any suggestions for how we should parameterize this? We could add another parameter to the function to pass in headers. Look to see if `body` is a special object with headers and content. Allow a `body` to be a function that we pass the `xhr` object to modify properties. Any other ideas?

It might also be a good time to take the default JSON parsing out of this function since some API endpoints might not necessarily return JSON. Maybe condition on the `responseType`